### DISCUSSION TITLE
[FD] Bug in Northstar FD mod

### DISCUSSION THREAD

**Discussion by t0mn8r** [R2Northstar/NorthstarMods#640] [General]
Hi,
When trying to run a dedicated FD server i get the following runtime error:
```
[20:07:00] [SCRIPT SV] [error] COMPILE ERROR Undefined variable "waveEvents"
[20:07:00] [SCRIPT SV] [error] gamemodes/_gamemode_fd.nut line [518] column [66]
```

```
from this line in _gamemode_fd.nut:
	for( int i = GetGlobalNetInt( "FD_currentWave" ); i < waveEvents.len(); i++ )
```
Also I believe there is a spelling mistake in the following line 809 in the same file:
		restetWaveEvents()
which I believe should read:
		resetWaveEvents()

No additional charge for the debugging...
📦 

I am hoping someone can fix that above bug which I believe is missing an array definition...

I think that a new fixed version should be created.

I am also looking for a way to do map rotations. I have been looking at these issues for 6 weeks now...

Any assistance is appreciated.

My Discord is t0mn8r#2095

email is tdesi@iinet.net.au

**Comment by GeckoEidechse** [comment]
Duplicate of #641

**Comment by t0mn8r** [comment]
Is anyone actually working on the fd mod ATM? It's got more bugs than the Amazon jungle...

**Reply by GeckoEidechse** [reply]
Almost as if Frontier Defense is unfinished and not released for a reason...

**Reply by F1F7Y** [reply]
> Almost as if Frontier Defense is unfinished and not released for a reason...

Sarcasm is unprofessional, I'll report this to Bob later today and suggest you be demoted from maintainer

**Reply by GeckoEidechse** [reply]
> > Almost as if Frontier Defense is unfinished and not released for a reason...
> 
> Sarcasm is unprofessional, I'll report this to Bob later today and suggest you be demoted from maintainer

Sarcasm is unprofessional, I'll report this to Bob later today and suggest you be demoted from maintainer

**Reply by pg9182** [reply]
Of course you will, won't you?
